server/conf/application.conf

```diff
@@ -246,7 +246,7 @@ play.http {
     # NOTE: this only sets when the browser will discard the cookie. Play will consider any
     # cookie value with a valid signature to be a valid session forever. To implement a server side session timeout,
     # you need to put a timestamp in the session and check it at regular intervals to possibly expire it.
-    #maxAge = 300
+    #maxAge = 300s
```

Comment:
(nonblocking) Can you add the link to the duration format so it's easier to find?

Reply:
Good idea thanks!